libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,31 @@
+import { loadScript, getConfig } from '../../utils/utils.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+export default async function init() {
+  if (!window.ost) {
+    await loadScript(`${base}/deps/offer-selector-tool.js`);
+  }
+
+  const ostCountry = 'US';
+  const ostLanguage = 'en';
+  const ostEnvironment = 'STAGE';
```

Comment:
i think stage should not be the default, as it's used for new feature (commerce side) only.

Reply:
Understood, what should be the default value instead?